Edit: Getting started (#79)
FalixNodes-Software/help-center: _posts/falix/general/2021-11-22-getting-started.md

---
layout: post
title:  "Getting Started"
categories: Falix
icon: <i class="fa-duotone fa-wand-magic-sparkles"></i>
tags: fxgeneral
permalink: /falix/general/getting-started/
---

# Getting Started

Welcome to FalixNodes! Here you can host game servers for free! To get started, sign up at the [Client Panel](https://client.falixnodes.net/auth/register). Make sure to read the [Privacy Policy](https://falixnodes.net/user-agreement/) and the [User Agreement](https://falixnodes.net/user-agreement/).

## Signing Up
There are two ways to sign up. You can either use your email and a password, or your Discord account.

### Registering With A Username And Password

1. Go to the [Register page](https://client.falixnodes.net/auth/register).
2. Fill in all the fields.
3. Click on "Register".

### Registering With Discord

1. Go to the [Register page](https://client.falixnodes.net/auth/register).

> 2. Scroll down and click on "Discord".

Reviewer: ```suggestion
2. Scroll down and click on "Discord". You will shortly be redirected to the Discord domain.
```

# Getting Started

Welcome to FalixNodes! Here you can host game servers for free! To get started, sign up at the [Client Panel](https://client.falixnodes.net/auth/register). Make sure to read the [Privacy Policy](https://falixnodes.net/user-agreement/) and the [User Agreement](https://falixnodes.net/user-agreement/).

## Signing Up

> There are two ways to sign up. You can either use a username and password or use a Discord account.

Reviewer: ```suggestion
There are two ways to sign up. You can either use your email and a password, or your Discord account.
```

1. Click on the account avatar at the top right of the page.
2. Click on "Settings".
3. Fill in all the fields under "Profile Settings".
4. Click on "Update".

## Creating A Server

> 1. Click on "Create Server". It should be in the navbar to your left.

Reviewer: ```suggestion
1. Click on "Create Server". It should be in the navbar on the left.
```

2. Scroll down and fill in all the fields given.
3. Click on "Create!".

## Logging Into The Game Panel

1. Go to the [Settings Page](https://client.falixnodes.net/profile/settings).
2. Make sure you have updated your details under the "Profile Settings" category.
3. Scroll down and click on "Change Password" under the "Danger" category.
4. Click on "Login To Panel". You should be able to see your login credentials.
5. To log in, go to the [Game Panel](https://panel.falixnodes.net) and fill in all your login credentials.

## Upgrading Your Server

Here in Falix, you can upgrade your server by earning coins then purchasing resources from the store. Follow the steps below:

### Getting Coins

> A guide to it can be found [here](https://help.falixnodes.net/falix/general/getting-coins/).

Reviewer: ```suggestion
A guide to this can be found [here](https://help.falixnodes.net/falix/general/getting-coins/).
```

### Buying Items From The Store

> A guide to it can be found [here](https://help.falixnodes.net/falix/general/buying-items-from-the-store/).

Reviewer: ```suggestion
A guide to this can be found [here](https://help.falixnodes.net/falix/general/buying-items-from-the-store/).
```

### Managing Your Server Allocations

> 1. Go to the [Client Panel's Main Page](https://client.falixnodes.net).

Reviewer: ```suggestion
1. Go to the [Client Panel](https://client.falixnodes.net).
```

2. Scroll down. You should see a list of your servers.
3. Click on the three buttons to its right, you should see a popup appear.

> 4. You can manage your server allocations here. You can also delete your server.

Reviewer: add an image of the popup, if you haven't already
